generate-types.js

```diff
@@ -15,10 +15,14 @@ async function main() {
     const protoDir = resolve('src', 'proto')
     const dcsProtoFile = resolve(protoDir, 'dcs', 'dcs.proto')
 
-    const generateTypes = `${bin} -I ${protoDir} -O ${generatedDir} --oneofs --grpcLib=@grpc/grpc-js ${dcsProtoFile}`
+    const args = ['-I', protoDir, '-O', generatedDir, '--oneofs', '--grpcLib=@grpc/grpc-js', dcsProtoFile]
 
-    exec(generateTypes)
+    await execFile(bin, args, { shell: true }) // shell:true makes this work on windows
```

Comment:
passing array args allows here is the fix i think. allowing node to handle spaces and encode properly. instead of us having to re-write the args encoding logic